messages/php/src/Id/FallbackUuidGenerator.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Cucumber\Messages\Id;
+
+final class FallbackUuidGenerator implements IdGenerator
+{
+    /**
+     * Inspired by
+     * @see https://github.com/clue/reactphp-clickhouse
```

Comment:
Would be better to link the UUIDv4 spec here.

Reply:
I literally stole the code with permission so wanted to link back

maybe I should co-authored-by instead? link to spec is a good idea 👍

Reply:
I'll just move this to the commit message ;)

Reply:
It's drifted enough that we don't need to credit; removed